content/en/security_platform/application_security/setup_and_configure.md

```diff
@@ -0,0 +1,190 @@
+---
+title: Setup and Configure Application Security Monitoring
+kind: documentation
+further_reading:
+- link: "/security_platform/application_security/"
+  tag: "Documentation"
+  text: "Monitoring Threats with Datadog Application Security"
+- link: "/security_platform/application_security/getting_started/"
+  tag: "Documentation"
+  text: "Getting Started Enabling Application Security Monitoring for Your Services"
+- link: "/security_platform/default_rules/#cat-application-security"
+  tag: "Documentation"
+  text: "Out-of-the-Box Application Security Rules"
+---
+
+## Compatibility
+
+{{< programming-lang-wrapper langs="java,dotnet,go,ruby,php,nodejs" >}}
+
+{{< programming-lang lang="java" >}}
+
+### Supported Java versions
+
+The Datadog library supports Java JRE 1.8 and higher of both Oracle JDK and OpenJDK, on the following architectures:
+- Linux (GNU) x86, x86-64
+- Alpine Linux (musl) x86, x86-64
+- macOS (Darwin) x86, x86-64
+- Windows (msvc) x86, x86-64
+
+Datadog does not officially support any early-access versions of Java. 
+
+### Supported frameworks
+
+| Framework Web Server    | Minimum Framework Version   | 
+| ----------------------- | --------------------------- |
+| Servlet                 | 2                           |
+| Spring                  | 3.1                         |
+
```

Comment:
```suggestion
Note: Many application servers are Servlet compatible and are automatically covered by AppSec, such as Websphere, Weblogic, and JBoss. Also, frameworks like Spring Boot inherently work because it usually uses a supported embedded application server (Tomcat/Jetty/Netty).
```

Reply:
```suggestion
**Note**: Many application servers are Servlet compatible and are supported by Application Security, such as WebSphere, WebLogic, and JBoss. Also, frameworks like Spring Boot are supported by virtue of using a supported embedded application server (such as Tomcat, Jetty, or Netty).
```